### DISCUSSION TITLE
Our Names

### DISCUSSION THREAD

**Discussion by Morbidous** [MNES-Emulator/MNES#17] [Q&A] [answered]
I guess we are using our real names in the repo. I didn't _really_ mean to do that, but it's kind of set in stone already, and I did it a long time ago already in the downunder UI. The more I think about it the more it bothers me, and I'm only drawing attention to it by talking about it. I might change new commits to be different but the damage is done and there's nothing to be done about it it's apparently just how git works. I'm hoping to alleviate some of the anxiety by being open about it but it might just backfire instead. oh well.

**Comment by jamieyello** [comment, marked as answer]
Hello, my name is Matthew James Cappel, now they will come after both of us (just don't call me Matt or Matthew please, I get enough of that irl). I assume the NSA already has me down pretty well at least, and I don't really make crazy internet enemies aside from a random spat.

If you want, we can take down the repo and put it all back up without older commits. It's not like building out basic structure is included very often anyway. There are also ways to [edit commit messages via cli](https://linuxize.com/post/change-git-commit-message/), that would be preferable.

**Comment by Morbidous** [comment]
yeah there are ways to rewrite history with different username/useremail but it's probably not worth the effort. just have to live with it.
